css_styles/styles.css

```diff
@@ -587,3 +614,306 @@ footer {
+
+  .mybio p {
+    width: 566px;
+    height: 123px;
+    font-family: "Inter", serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+    flex: none;
+    order: 2;
+    flex-grow: 0;
+  }
+
+  .myworks {
+    grid-template-columns: 1fr 1fr;
+    padding: 20px 24px;
+  }
+
+  #work-headline {
+    border: none;
+    line-height: 70px;
+  }
+
+  .line {
+    width: 50%;
+    border: 1px solid black;
+  }
+
+  .multi-storybg {
+    height: 470px;
+    padding: 0;
+    margin: 0;
+    width: 50%;
+  }
+
+  .div-myworks {
+    height: auto;
+  }
+
+  .projects-section {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+
+  #about-me {
+    background-image: url(/images/aboutme-desktop.png);
+    background-repeat: no-repeat;
+    background-position-y: center;
+  }
+
+  #forme {
+    font-family: 'Crete Round', sans-serif;
+    font-size: 72px;
+    font-weight: 400;
+    line-height: 82px;
+    letter-spacing: 0;
+    text-align: left;
+  }
+
+  #about-me p {
+    font-family: "Inter", sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 23px;
+    letter-spacing: 0;
+    text-align: left;
+  }
+
+  .resume-btn {
+    bottom: -10%;
+  }
+
+  #proj1:hover {
+    background-image: url(/images/desktop\ project-1.png);
+    cursor: pointer;
+  }
+
+  .project-card:hover {
+    display: none;
+    transition: all 0.2s ease-in-out;
+  }
+
+  #proj2:hover {
+    background-image: url(/images/desktop\ project-2.png);
+    cursor: pointer;
+  }
+
+  #proj3:hover {
+    background-image: url(/images/desktop-project-3.png);
+    cursor: pointer;
+  }
+
+  #proj1:hover > .projbtn {
+    display: block;
+  }
+
+  #proj2:hover > .projbtn {
+    display: block;
+  }
+
+  #proj3:hover > .projbtn {
+    display: block;
+  }
+
+  .works-title {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .work-image {
+    background-image: url(/images/myworks-desktop.png);
+    background-size: cover;
+    margin: 0;
+    width: 684px;
+    height: 448px;
+  }
+
+  #pro-languages {
+    display: grid;
+    flex-direction: column;
+    width: 100vw;
+    grid-template-columns: 1fr 1fr 1fr;
+    justify-content: center;
+    justify-items: center;
+  }
+
+  .dt-list {
+    flex-direction: column;
+    align-items: flex-start;
+    width: 100%;
+  }
+
+  .div-skills {
+    width: auto;
+    height: 263px;
+  }
+
+  .my-thumbnail {
+    position: relative;
+  }
+
+  .rectangle {
+    position: absolute;
+    left: -30px;
+  }
+
+  .square {
+    position: absolute;
+    left: -32px;
+  }
+
+  .circle {
+    position: absolute;
+    left: 18px;
+  }
+
+  #for-getintouch {
+    display: flex;
+    width: 100%;
+    height: 100%;
+    padding: 0;
+  }
+
+  .on-mobile {
+    display: none;
+  }
+
+  .on-desktop {
+    display: block;
+  }
+
+  #myform {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    margin: 100px 158px 154px 0;
+  }
```

Comment:
Kindly make sure you follow the Figma design as I mentioned above. I suggest you stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

>### Your design
>
><img width="1440" alt="Screenshot 2023-05-05 at 2 52 59 PM" src="https://user-images.githubusercontent.com/91237525/236547928-58f6ed2b-6286-469d-a030-cd45d52e0c0c.png">
>
>### The Figma Design
>
><img width="737" alt="Screenshot 2023-05-05 at 3 06 50 PM" src="https://user-images.githubusercontent.com/91237525/236548326-a4062be2-a849-41df-a884-7abacada0fa2.png">

Reply:
I have implemented the changes mentioned by you dear sir.

Reply:
Please remove the background color to make your form look same as the template 
![temp2 contact color](https://user-images.githubusercontent.com/59707859/236624584-c6b97db2-cf22-428f-957c-e798d6e6b643.PNG)